tests/_internal/test_write_files.py

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (C) 2020 Arm Mbed. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+import pathlib
+import tempfile
+from unittest import TestCase
+
+from mbed_build._internal.write_files import write_file
+from mbed_build.exceptions import InvalidExportOutputDirectory
+
+
+class TestWriteFile(TestCase):
+    def test_writes_content_to_file(self):
+        with tempfile.TemporaryDirectory() as directory:
+            content = "Some rendered content"
+            export_path = pathlib.Path(directory, "output")
+            file_name = "some_file.txt"
+
+            write_file(export_path, file_name, content)
+
+            created_file = pathlib.Path(export_path, pathlib.Path(export_path, file_name))
+            self.assertTrue(created_file.is_file())
```

Comment:
A bit redundant, since we're reading contents in the next line.

Reply:
That is true :)

Reply:
44f0f4f